pages/accountLists/[accountListId]/tools/appeals/AppealsWrapper.tsx

```diff
@@ -81,24 +84,31 @@ export const AppealsWrapper: React.FC<Props> = ({ children }) => {
     });
   };
 
+  const setActiveFilters: AppealsContextProps['setActiveFilters'] = (value) => {
+    const filters = typeof value === 'function' ? value(activeFilters) : value;
```

Comment:
It's just because `Dispatch<SetStateAction<_>>` means that you can pass a value or a function to the setter. But I don't think we're actually passing in a function. I could have also changed the type of `setActiveFilters` to `(filters: ContactFilterSetInput & TaskFilterSetInput) => void`. I might actually do that.